crates/matrix-sdk-common/Cargo.toml

```diff
@@ -46,6 +46,7 @@ assert_matches = { workspace = true }
 proptest = { workspace = true }
 matrix-sdk-test-macros = { path = "../../testing/matrix-sdk-test-macros" }
 wasm-bindgen-test = { workspace = true }
+insta = { version = "1.41.1", features = ["json"] }
```

Comment:
Let's put this right away as a workspace dependency, it's very likely that we're going to use that in multiple crates.